submissions/mustbefail/tiny-js-world/index.js

```diff
@@ -0,0 +1,89 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const cat = {
+  species: 'cat',
+  name: 'Prokhor',
+  hands: 0,
+  legs: 4,
+  saying: 'Meow!',
+  gender: 'male',
+  friends: 'June, Maki',
+};
+const catWoman = {
+  species: 'cat',
+  name: 'June',
+  hands: 0,
+  legs: 4,
+  saying: cat.saying,
```

Comment:
What will be if `cat` obj will be declared after `catWoman` ?

Reply:
Error calling before initialization. Need to move a property **saying** to a shared object?

Reply:
No, shared obj is not necessary here

Reply:
BTW, your idea is good. This is not a blocker so i can approve this pr